core/src/main/javascript/app/components/Badge.js

```diff
@@ -72,7 +72,9 @@ const styles = {
     borderRadius: "2px",
     padding: "0 .5em",
     maxHeight: "18px",
-    textAlign: "center"
+    textAlign: "center",
+    marginRight: "1px",
+    userSelect: "none"
```

Comment:
There is a general alignement problem with thses badge. Occuring again in this new screen. Time to fix it? For example with `verticalAlign: "middle"`

Reply:
I think we should address this in another PR.